spec/models/manifest_spec.rb

```diff
@@ -148,4 +148,19 @@
       end
     end
   end
+
+  describe '#is_public?' do
```

Comment:
the English student in me always likes the explicit verb but I see that Ruby conventions tend toward brevity.

http://programmers.stackexchange.com/questions/232213/should-i-always-use-is-as-prefix-for-boolean-variables

I find this more readable, personally:

```
if thing.is_foobar?
```

than this:
```
if thing.foobar?
```

because the former reads like an actual sentence, and clarifies its being-ness rather than its acting-ness. The `?` indicates that the return value is a boolean, but does not necessarily imply that the method is idempotent.

ok, spiel over. if you don't care much, let's leave as is.